src/app/content/components/Page/contentDOMTransformations.ts

```diff
@@ -125,3 +126,47 @@ function prefixResources(rootEl: HTMLElement) {
     (el) => el.src = REACT_APP_ARCHIVE_URL + el.getAttribute('src')
   );
 }
+
+function moveFootnotes(document: Document, rootEl: HTMLElement, intl: IntlShape) {
+  const footnotes = document.querySelectorAll('[role=\'doc-footnote\']');
+  if (!footnotes.length) { return; }
+
+  const title = intl.formatMessage({id: 'i18n:content:footnotes:title'});
+  const header = document.createElement('h3');
+  header.setAttribute('data-type', 'footnote-refs-title');
+  header.innerHTML = title;
+
+  const container = document.createElement('div');
+  container.setAttribute('data-type', 'footnote-refs');
+  container.appendChild(header);
+
+  const list = document.createElement('ul');
+  list.setAttribute('data-list-type', 'bulleted');
+  list.setAttribute('data-bullet-style', 'none');
+
+  for (const [index, footnote] of Array.from(footnotes).entries()) {
+    const counter = index + 1;
+
+    const item = document.createElement('li');
+    item.setAttribute('id', `footnote${counter}`);
```

Comment:
Since there is no information in an issue about how this should look like, I've replicated current design, for example: https://rex-web.herokuapp.com/books/biology-2e/pages/45-4-population-dynamics-and-regulation#72203

I found it by using Elefind: https://content-finder.herokuapp.com/?book=Biology_2e&element=%5Bdata-type%3D%22footnote-ref-link%22%5D